src/main/java/se/sundsvall/casestatus/service/CaseStatusService.java

```diff
@@ -199,14 +201,14 @@ List<CaseStatusResponse> filterResponses(final List<CaseStatusResponse> response
 	}
 
 	public List<CaseStatusResponse> getErrandStatuses(final String municipalityId, final String propertyDesignation, final String errandNumber) {
-		if (StringUtils.isNotBlank(propertyDesignation) && StringUtils.isNotBlank(errandNumber)) {
+		if (isNotBlank(propertyDesignation) && isNotBlank(errandNumber)) {
```

Comment:
`StringUtils.hasText(str)` does the same but does not negate the expression. 
I.e. it might increase readability